frontend/src/features/mapSideBar/tabs/SectionField.tsx

```diff
@@ -0,0 +1,37 @@
+import { Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+
+interface ISectionFieldProps {
+  label: string;
+  content: string[];
+  isMultiLine?: boolean;
+}
+
+export const SectionField: React.FunctionComponent<ISectionFieldProps> = props => {
+  return (
+    <Row className="pb-2">
+      <Col xs="4">
+        <StyledFieldLabel>{props.label}:</StyledFieldLabel>
+      </Col>
+      <Col>
+        {props.content.map((item, index) =>
+          props.isMultiLine === true ? (
+            <Row key={`label ${props.label} ${index}`}>
+              <Col>
+                <span>{item}</span>
+              </Col>
+            </Row>
+          ) : (
+            <span key={`label ${props.label} ${index}`} className="mr-2">
+              {item}
+            </span>
+          ),
+        )}
+      </Col>
+    </Row>
+  );
+};
+
+const StyledFieldLabel = styled.div`
```

Comment:
this should probably be a label not a div.

Reply:
Also would be nice to be able to pass the 'for' as a prop so that jests *ByLabel methods work. Also this helps with accessibility.

Reply:
another option would be to use a Form.Group with a controlId as that ensures the child Form.Label and Form.Control are linked appropriately.

Reply:
I like the label instead of div, the form group could be done later if needed

Reply:
I opted to leave the div as is. If there is a need for inputs to match the label we could pass a isForm or something like that so it changes dynamicaly